core/trino-main/src/test/java/io/trino/server/ui/TestWebUi.java

```diff
@@ -677,8 +682,32 @@ public void testOAuth2AuthenticatorWithoutOpenIdScope()
                         .setBinding()
                         .toInstance(oauthClient))
                 .build()) {
-            HttpServerInfo httpServerInfo = server.getInstance(Key.get(HttpServerInfo.class));
-            assertAuth2Authentication(httpServerInfo, oauthClient.getAccessToken());
+            assertAuth2Authentication(server, oauthClient.getAccessToken(), false);
+        }
+        finally {
+            jwkServer.stop();
+        }
+    }
+
+    @Test
+    public void testOAuth2AuthenticatorWithRefreshToken()
+            throws Exception
+    {
+        OAuth2ClientStub oauthClient = new OAuth2ClientStub(false);
+        TestingHttpServer jwkServer = createTestingJwkServer();
```

Comment:
Can we use try-with-resource for `jwkServer`

Reply:
But `TestingHttpServer` is not `Closable`